wireapp/wire-ios-mono
# fix: video toggle disabled in MLS calls - WPB-6204

<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-6204" title="WPB-6204" target="_blank"><img alt="Bug" src="https://wearezeta.atlassian.net/rest/api/2/universal_avatar/view/type/issuetype/avatar/10803?size=medium" />WPB-6204</a>  [iOS] MLS: Cannot turn on camera when joining video call on bund-next
  </td></table>
  <br />
 












----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

### Issues

On a MLS conference calls, sometimes the video button is disabled

### Causes (Optional)

When receiving the call, we set the wrong value of `isConferenceCall`  to the callSnapshot. We check only for conference value and not mlsConference. This results on caller being able to enable the video but not receivers.

### Solutions

Use helper method from AVSIdentiferType to get the right value of `isConferenceCall`

### Testing

#### Test Coverage (Optional)


#### How to Test

- MLS Call with multiple people
- answer on iphone
- you should be able to enable video
